### PR TITLE
Created popup project window

### PR DESCRIPTION
**Pull request by mikemtzp**
## For this feature we:

- Created Javascript code that dynamically generates a card for each project's information added within an object in an array.
- Created Javascript code that dynamically generates a pop-up window for each project to see further information.
- Styled both mobile and desktop pop-up windows.
- Added Javascript event so you can open and close the pop-up window with a click over an 'X' icon as pleased.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -4,6 +4,172 @@
...
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  z-index: 20;
+  background: #fff;
+  gap: 15px;
+  padding: 3% 5%;
+}
```
Comment (CollinsTatang) [inline style.css:42]: Hi @mikemtzp 
Your project is great but you have to make some required changes,

- [x] Please the pop-up is not responsive to mobile view.
### **Take A Look**
screenpop

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @mikemtzp, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mikemtzp** [comment]
Hi @CollinsTatang!
 
Thank you for the review!

The popup resizes correctly resizes on my side:

image

I also tried some of the mobile views inside of the browser inspector and they also look fine.

image
image
image
image

In addition, I see you have your size as 375x411. I really think the mobile size would be with other sizes.
Could you explain a little more about what exactly I need to adjust?

**Review by m05e5 — APPROVED** [review]
# Hi @mikemtzp , 

### Status: Approved ✔️
goodjob


:+1: You have done all the required changes asked by the previous reviewer. Well Done.

Your project is complete! 
Congratulations! 🎉 


**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_m05e5_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by CollinsTatang** [comment]
> Hi @CollinsTatang!
> 
> Thank you for the review!
> 
> The popup resizes correctly resizes on my side:
> 
> image
> 
> I also tried some of the mobile views inside of the browser inspector and they also look fine.
> 
> image image image image
> 
> In addition, I see you have your size as 375x411. I really think the mobile size would be with other sizes. Could you explain a little more about what exactly I need to adjust?

Hi @mikemtzp 
Your work looks perfect now, Congratulations

**Pull request merged**